components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -195,7 +195,7 @@
   // ------------------------------------------------------------------------
   // Behavior of the quick search box
   // ------------------------------------------------------------------------
-  var offsetTop = $('patient-record-actions') &amp;&amp; $('patient-record-actions').getHeight() || 0;
+  var offsetTop = 0;
```

Comment:
The previous value for this line added a vertical offset between the top of the viewport and the "Quick phenotype search" box, with a height equal to the patient actions toolbar. Was there a reason to remove this offset?
![p0000001___phenotype_database](https://cloud.githubusercontent.com/assets/4006062/15580415/ac540658-2336-11e6-8672-828c944d9b94.jpg)